countdown.js

```diff
@@ -40,26 +41,31 @@ const countdown = function(_config) {
     // Find the distance between now an the count down date
     const distance = countDownDate - now;
 
+    if (distance < 0) {
+      cdwn(".countdown").innerHTML = `<span class="expired countdown__expired">${_config.expired ? _config.expired : 'expired'}</span>`;
+      clearInterval(animationFrame)
+      return false;
+    }
+
     // Time calculations for days, hours, minutes and seconds
     const days = Math.floor(distance / (1000 * 60 * 60 * 24));
     const hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
     const minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
     const seconds = Math.floor((distance % (1000 * 60)) / 1000);
 
-    requestAnimationFrame(updateTime);
+    //requestAnimationFrame(updateTime);
 
-    $(_config.target+' .day .num').innerHTML = addZero(days);
-    $(_config.target+' .hour .num').innerHTML = addZero(hours);
-    $(_config.target+' .min .num').innerHTML = addZero(minutes);
-    $(_config.target+' .sec .num').innerHTML = addZero(seconds);
+    cdwn(_config.target+' .day .num').innerHTML = addZero(days);
+    cdwn(_config.target+' .hour .num').innerHTML = addZero(hours);
+    cdwn(_config.target+' .min .num').innerHTML = addZero(minutes);
+    cdwn(_config.target+' .sec .num').innerHTML = addZero(seconds);
 
     // If the count down is over, write some text
-    if (distance < 0) {
-      $(".countdown").innerHTML = "EXPIRED";
-    }
-  }
 
-  updateTime();
+  }
+  //start
+  updateTime()
+  animationFrame = setInterval( updateTime, 1000)
```

Comment:
I am against changing to `setInterval` unless you have a strong reason to support the change. Anyway I'd suggest you to use your own fork.